fw/http.h

```diff
@@ -205,11 +206,12 @@ typedef enum {
 	TFW_HTTP_HDR_X_FORWARDED_FOR,
 	TFW_HTTP_HDR_KEEP_ALIVE,
 	TFW_HTTP_HDR_TRANSFER_ENCODING,
+	TFW_HTTP_HDR_X_TEMPESTA_CACHE,
 
 	/* Start of list of generic (raw) headers. */
 	TFW_HTTP_HDR_RAW,
 
-	TFW_HTTP_HDR_NUM	= 16,
+	TFW_HTTP_HDR_NUM	= 17,
```

Comment:
We don't need the constant, `16` or `17`, the compiler will assign the constant to the last enum member automatically.